apps/openmw/mwgui/windowmanagerimp.hpp

```diff
@@ -361,6 +361,8 @@ namespace MWGui
 
     virtual void pinWindow (MWGui::GuiWindow window);
 
+    void updatePinnedWindows ();
```

Comment:
I mean that the updatePinnedWindows declaration should be moved from public: to private: section since it's only used internally.